consul/src/main/scala/io/buoyant/consul/v1/ConsistencyMode.scala

```diff
@@ -0,0 +1,37 @@
+package io.buoyant.consul.v1
+
+import com.fasterxml.jackson.core.{JsonGenerator, JsonParser}
+import com.fasterxml.jackson.databind.{DeserializationContext, SerializerProvider}
+import io.buoyant.config.{ConfigDeserializer, ConfigSerializer}
+
+sealed trait ConsistencyMode
+
+object ConsistencyMode {
+
+  case object Default extends ConsistencyMode
+  case object Stale extends ConsistencyMode
+  case object Consistent extends ConsistencyMode
```

Comment:
after reading up, it seems like the only real value of `case object` is that it generates a "pretty" `toString`.  not a big deal one way or the other.